utilities/backupable/backupable_db.cc

```diff
@@ -1381,6 +1432,22 @@ Status BackupEngineImpl::CopyOrCreateFile(
   if (s.ok()) {
     s = dest_writer->Close();
   }
+  if (s.ok() && UseNewNaming()) {
+    // When copying SST files and using db_session_id in the name,
+    // try to get DB identities
+    // Note that when CopyOrCreateFile() is called while restoring, this block
+    // of code is not executed as for now we do not use session_id to verify
```

Comment:
Sorry I didn't understand. Isn't `src` still nonempty in the restore case? I thought it's only empty in case we are creating a file, like in the CURRENT file case.

Reply:
Right, the comment is not properly written :P. I just wanted to stress the fact that the ids are not used for any purposes in restoring the backups. (Will rewrite it properly :)